content/blog/2018-08-21-birds-radolfzell.md

```diff
@@ -0,0 +1,528 @@
+  knitr::kable()
+```
+
+| scientific\_name    | common\_name      | approved | reviewed | reason                    |
+|:--------------------|:------------------|:---------|:---------|:--------------------------|
+| Cygnus atratus      | Black Swan        | FALSE    | TRUE     | Species-Introduced/Exotic |
+| Cygnus atratus      | Black Swan        | FALSE    | TRUE     | Species-Introduced/Exotic |
+| Cygnus atratus      | Black Swan        | FALSE    | TRUE     | Species-Introduced/Exotic |
+| Oxyura leucocephala | White-headed Duck | FALSE    | TRUE     | Species-Introduced/Exotic |
+| Mareca sibilatrix   | Chiloe Wigeon     | FALSE    | TRUE     | Species-Introduced/Exotic |
+
+Black Swans [are mostly present in Australia, imported and escaped in a
+few other places](https://en.wikipedia.org/wiki/Black_swan) but eBird
+mostly doesn’t accept the entry of exotic species [although it’s
+debated](https://help.ebird.org/customer/portal/questions/5390335-exclusion-of-not-native-birds-in-lists).
+In any case, eBird’s control of the data entered is quite admirable.
+
+#### Who observed birds?
+
+In [one of his latest blog
+posts](https://recology.info/2018/06/butte-county/) Scott Chamberlain
+mentioned the legendary Lowell Ahart, super plant collector in Butte
+County, California. Does the county of Constance have a super birder?
+
+``` r
+(first_birder <- ebd %>%
+  dplyr::count(observer_id) %>%
+  dplyr::arrange(- n) %>%
+  head(n = 1) )
+```
+
+    ## # A tibble: 1 x 2
+    ##   observer_id     n
+    ##   <chr>       <int>
+    ## 1 obsr457108   3551
+
+``` r
+(proportion <- round(first_birder$n/nrow(ebd),
+                    digits = 2))
+```
+
+    ## [1] 0.35
+
+Wow, that person made 35% of eBird observations in the county! The EBD
+no longer provides names (consequence of the [EU General Data Protection
+Regulation](https://www.eugdpr.org/)) but from the checklist ID one can
+get access to the checklist page e.g [this
+one](https://ebird.org/view/checklist/S42391392) where the name of the
+observer is present. The super birder of the County of Constance is
+[Antonio Anta Bink](https://ebird.org/profile/NDU3MTA4/DE).
+
+### Conclusion
+
+#### R packages for occurrence data
+
+In this post we gave a rough view of what birds are present in the
+county around Radolfzell: Eurasian Blackbirds, Carrion Crows, Great
+Tits… but not Black Swans in eBird’s data. We mostly illustrated the use
+of two R packages accessing eBird’s data:
+
+-   `auk` for processing the gigantic whole eBird’s dataset.
+
+-   `rebird` for getting access to recent data via an API. `rebird` is
+    part of a larger collection of packages for occurrence data within
+    rOpenSci’s suite, with `spocc` being an umbrella package accessing
+    several data sources; `scrubr` an helper for cleaning data obtained
+    this way; and `mapr` an utility package for mapping such data.
+
+Explore these packages, and more rOpenSci’s packages, by checking out
```

Comment:
too many "packages"

Explore these rOpenSci packages, and more by checking out our packages page!